src/pages/index/components/ScheduleTable/ClassCard.vue

```diff
@@ -7,6 +7,9 @@
     <div class="venue" v-else>
       <span class="venue-at">@</span>{{ $store.getters.extra(`${course.courseId}-${course.teacherId}`).venue }}
     </div>
+    <div class="extra" v-if="course.fortnight || course.lab">
+      {{ course.fortnight }} {{course.lab}}
```

Comment:
```suggestion
      {{ course.fortnight }} {{ course.lab }}
```
style: add spaces in template interpolation